proxy/envoy/discovery.go

```diff
@@ -151,6 +151,12 @@ type hosts struct {
 type host struct {
 	Address string `json:"ip_address"`
 	Port    int    `json:"port"`
+	Tags    tags   `json:"tags,omitempty"`
```

Comment:
Perhaps we want *tags instead of tags?  I played around with this last week (trying to write my own, non-Istio SDS) and the _omitempty_ didn't do what I wanted unless tags was a *.